README.md

```diff
@@ -1,49 +1,97 @@
-# Software Stack
 
 ![Ubuntu](https://github.com/noaa-emc/hpc-stack/workflows/Build%20Ubuntu/badge.svg)
 ![macOS](https://github.com/noaa-emc/hpc-stack/workflows/Build%20macOS/badge.svg)
 
-This repository provides a unified, shell script based build system for building commonly used software stack.
+# hpc-stack
+
+This repository provides a unified, shell script based build system
+for building the UFS weather model software stack. This is run on NOAA
```

Comment:
This software stack builds the software stack for more than just the UFS weather model (and hence the name hpc-stack and not ufs-stack).

Reply:
Awesome! What should I add? JEDI? What else?

Reply:
how about this ...

... for building the software stack used in NOAA applications such as the UFS model, JEDI-based data assimilation apps, etc.

Reply:
Does JEDI have a website?

Not keen on "etc." What else other than UFS and JEDI, if anything?

Reply:
UFS and JEDI are fine.
I didn't see a UFS website other than the github page.
JEDI RTD is at: https://jointcenterforsatellitedataassimilation-jedi-docs.readthedocs-hosted.com/en/latest/

Reply:
OK, I've edited - take a look at the new version.